source/uwp/AdaptiveCardTestApp/Expected/v1.0.sampleDifferentFont.Tests.Action.Title.json

```diff
@@ -0,0 +1 @@
+{"HostConfigHash":"5284388","CardHash":"cb85211","Error":"Parsing card failed"}
```

Comment:
looks like this and two other files below it are missing images

Reply:
These cards fail to parse (by design) because they have invalid card json (this one is missing action titles and the next one is missing version). As a result, there's no rendered card. The "Error":"Parsing card failed" in this file will tell the test app that the failure is expected.

---
In reply to: [178395259](https://github.com/Microsoft/AdaptiveCards/pull/1321#discussion_r178395259) [](ancestors = 178395259)